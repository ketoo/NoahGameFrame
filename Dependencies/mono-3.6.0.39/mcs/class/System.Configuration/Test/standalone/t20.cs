using System;
using System.Configuration;

class T1
{
	static void Main(string[] args)
	{
		try
		{
			Configuration config = ConfigurationManager.OpenExeConfiguration (ConfigurationUserLevel.None);
			AppSettingsSection sect = (AppSettingsSection)config.GetSection("appSettings");

			Console.WriteLine (sect.SectionInformation.GetRawXml ());

			foreach (string key in sect.Settings.AllKeys) {
				Console.WriteLine ("settings[{0}] = {1}", sect.Settings[key].Key, sect.Settings[key].Value);
			}
		}
		catch (Exception e)
		{
			Console.WriteLine ("Exception raised: {0}", e);
		}
	}
}
