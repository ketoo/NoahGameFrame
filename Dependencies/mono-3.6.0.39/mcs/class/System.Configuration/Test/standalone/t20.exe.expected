<appSettings>
		<add key="aloha" value="aloha" />
	</appSettings>
settings[aloha] = aloha
